src/app/services/wallet.service.ts

```diff
@@ -439,7 +443,7 @@ export class WalletService {
   }
 
   private resetBalancesTimerOptions(hasPendingTxs: boolean) {
-    this.intervalTime = (hasPendingTxs ? 20 : 60) * 1000;
+    this.intervalTime = (hasPendingTxs ? 20 : 300) * 1000;
```

Comment:
Do not change this or the label with how many minutes have passed since the last update will stop updating the time correctly.